android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1858,6 +1860,7 @@ public void onTransportConnected(final TransportRecord record) {
             //the developer can use this pendingIntent to start their SdlService from the context of
             //the active RouterService
             Intent pending = new Intent();
+            pending.setPackage(SDL_PACKAGE);
```

Comment:
I think it is better to use the actual package of the currently running service than one that is generic one.

```suggestion
            pending.setPackage(getPackageName());
```